lib/tests/modules.sh

```diff
@@ -208,8 +208,8 @@ checkConfigOutput the-file.nix config.submodule.internalFiles.0 ./submoduleFiles
 # Check that disabledModules works recursively and correctly
 checkConfigOutput '^true$' config.enable ./disable-recursive/main.nix
 checkConfigOutput '^true$' config.enable ./disable-recursive/{main.nix,disable-foo.nix}
-checkConfigOutput '^true$' config.enable ./disable-recursive/{main.nix,disable-bar.nix}
-checkConfigError 'The option .* does not exist. Definition values:\n\s*- In .*: true' config.enable ./disable-recursive/{main.nix,disable-foo.nix,disable-bar.nix}
+checkConfigOutput '^true$' config.enable ./disable-recursive/{main.nix,disable-bar-via-stringpath.nix}
+checkConfigError 'The option .* does not exist. Definition values:\n\s*- In .*: true' config.enable ./disable-recursive/{main.nix,disable-foo.nix,disable-bar-via-stringpath.nix}
```

Comment:
This is not the right place to test this. Instead I suggest adding a test like this after line 132:

```
checkConfigOutput '^false$' "$@" ./v
```

and creating the file `modules/disable-string-define-enable.nix`:
```nix
{ lib, ... }:

{
  disabledModules = [ (toString ./define-enable.nix) ];
}
```